### PR TITLE
4716 card reveal refactoring

### PR DESCRIPTION
**Pull request by Dr-Java**

### CODE REVIEW

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java [hunk 1]
```diff
@@ -135,7 +136,8 @@ public void darkPaginationTest() {
     }
 
     @Test(description = "Test checks color pagination components: size, values, theme, "
-        + "color of active and inactive items")
+            + "color of active and inactive items",
+            enabled = false)
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java:140]: Sorry, please, enable as all tests are fixed :)

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java [hunk 2]
```diff
@@ -100,7 +100,8 @@ public void lengthPaginationTest() {
         lengthPagination.is().atEnd();
     }
 
-    @Test(description = "Test checks visible pagination components")
+    @Test(description = "Test checks visible pagination components",
+            enabled = false)
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java:104]: Sorry, please, enable as all tests are fixed :)

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java [hunk 3]
```diff
@@ -116,7 +117,7 @@ public void totalVisiblePaginationTest() {
             actualButtonsFromEndToStart.add(totalVisiblePagination.selected());
         }
         jdiAssert(actualButtonsFromEndToStart, equalTo(asList(
-            "15", "14", "13", "12", "11", "10", "9", "8", "7", "6", "5", "4", "3", "2", "1"
+                "15", "14", "13", "12", "11", "10", "9", "8", "7", "6", "5", "4", "3", "2", "1"
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java:120]: what is this change for :)

File: jdi-light-vuetify-tests/src/main/java/io/github/com/custom/cards/RevealCard.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        return primaryText;
+    }
+
+    public Button getLearnMoreButton() {
+        return learnMoreButton;
+    }
+
+    public Card getReveal() {
+        return new Card().setCore(Card.class, this.find("//div[contains(@class, 'transition')]"));
+    }
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/main/java/io/github/com/custom/cards/RevealCard.java:31]: why do you choose 'transition' for locator? 'transition' is a very tricky thing)))

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java (hunk 3 above)
Comment (Dr-Java) [reply jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/PaginationTests.java:120]: Just formatting)

File: jdi-light-vuetify-tests/src/main/java/io/github/com/custom/cards/RevealCard.java [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    @UI("button")
+    private Button learnMoreButton;
+
+    @Override
+    public Text title() {
+        return new Text().setCore(Text.class, core().find(".text-h4"));
+    }
+
+    public Text getPrimaryText() {
+        return primaryText;
```
Comment (MayaElf) [inline jdi-light-vuetify-tests/src/main/java/io/github/com/custom/cards/RevealCard.java:22]: we need to clarify issue with naming concerning using 'get'

**Review by MayaElf — CHANGES_REQUESTED** [review]

**Review by MayaElf — APPROVED** [review]

**Review by vklonin — APPROVED** [review]

**Pull request merged**